src/VisualStudio/Core/Def/InheritanceMargin/InheritanceMarginTaggerProvider.cs

```diff
@@ -150,5 +150,8 @@ protected override IEnumerable<SnapshotSpan> GetSpansToTag(ITextView? textView,
                     new InheritanceMarginTag(lineNumber, membersOnTheLineArray)));
             }
         }
+
+        protected override bool Equals(InheritanceMarginTag tag1, InheritanceMarginTag tag2)
+            => tag1.Equals(tag2);
```

Comment:
Why is this necessary? Wasn't the default still to call object.Equals which you already called?

Reply:
no.  there is no default.  i wanted an override in all cases to force consideration of the correct semantics :)